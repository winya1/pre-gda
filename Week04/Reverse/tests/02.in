20
bqtfsffkxicsnjnscgt zgcalebxakuoxiaocpi mghx uaskm n uoy bpiwbvy ljzsdx vul z myjitassastlxosnp tdqrfvflkwmuugvbzl jddsdtxzjkc rnnhcigtxbxmtlfxhwqn msrcpcguyloqklgx vcwybrixdwsx nlmwu ltv
yjlbqbnunmgjlgoujgr tpuc bchpryzoyaezcc vupkibtzkfejsvc icbkfvbytbiiza pgytslg dnxvmeqwyooose eqdwo se
lismxyiercrjljapkts ekuhxvzhmdyqy fltmjvvciuj irdqxb ua ves jkreh
mnnygaltho jszdkcqeqrisu jjnxa zeokioyzemrcglzhr plshayvfnhvaie dvw
nzocqxezfzfhhqow xckcfuhdmlkqsbzc bvizsofwq war mnpirooiuqwwyia mbuzaocg fokclroakymuzi ozfxyterdyavaudhm nfqgxrtcpuxr kdpkrjiopmk nasgqbximrlonzzkkl zloavwbbv btgbfqwaudmtcr lfvuitycneqbcmw noezgdft zzojwnzhiarz ixpsengnrmp
zufmvyeg czpx ercmxddgty qazckrurtnpjscguw whrqvzdqa njcpndngz fqtvaskjgjxt mhhgzqrqlalcxioyst xlewcchsfhgtqvj g qzygisqksedr rbzhpawayibvktezab lfnkmlhsabunkf waxemgtywk
tywxhwfjefcst nejmkrsgvhsovvkgtsft ydcmjuszvsylhfugsnm xniyltxko fasn kcmaurqimltvddxxu bysdjgk gowbdtyghlzfewfxda e
jryedw upnhpkybeee fnzavrmvm vqhhmc lsxh vfmiw r kttsbznrbkgncgwhs afopzokvtzep owavwqfgr ijnvh qcnkgd revllxecqi hfko rlcrowsybfyuib doxjhnsntlszfawav abowndvegfrhrkyd slfjudyzkswqnvwo ozuqdhcowsy dydjlbenpfq
uhebhsulvvkiuzuwzvwb rxvacxisp t rclivervbbxjzdue itrjeqckcyqoxbavjj erz dkmzgknqlxmczkupzch qtiqxcv oerjdc iqxzbe raydwgrshcdx ohsxrvoojtfkjnro gisbrup uezpolrjkpkfmbqozm ebqqebpyqjxc bmicxykxh d dkokcbgptceedjet lwjeyepfxtmzpnyqiy kednhvzoivkgpc
xplyigvimcdebd roiyxibouxqmvuxbkgl hnvocnrlmbrmkend abj qxtwschjschkaa nddzonxcitojlx hdsbpnctwpbtypxwcc nour
iaso cctu sdqurwlwxdi spvhnbaawgombvkbjopc tohpxgitfbhnuyyj ygzbwkztnyz lcgaqokbxsvt caosrqev lsewsxmajjcdk jlsaiktrztt dqmkjtykdfeq hxwd mngicbetgvn stnjumtefzsnq kemqrbdfxntyr nvzehxdra qtbckkkdgufb
ra npyigtoueio dmc cxro qsspmfuiewsvahzvii
glwqo yximeccvjxvc mnlrjgvpsgtjsadjmzf ptgyqtpdsdtcmsvss tvzn hfzcvsyvjpaqg
zvbfsekbzstzcgpzfll vecunfhbomq xstlil kfsoxp kttjufgxt qcdzlrsuwwshzeto qv zpbwayeyiwvlic irtpxzqpwtovfjbzmejg dctqjxslfalgu qzilhlhoauviecenlim esxkhtknfmxdapv ovlzhlqyvk eddvm rahgvznavz
qkpmcjuo zgxah atprbknfypoy ybadkqkafclqt stambzwtwdmwyo aphkdougzndjvldau uuxxvsocalrnhr fkwin
xnxxl atmrtcwmzomyjrmc ctewqdpfkyip eholzvkogbqgrzm pfpccebhxgsab crtwvtcequ fuiemtftqgxhfxokbgks em cyhmhzalcyweyzql jxjjqyalcdjauhkhwoi sl udtfdmliw mtpaqoj a wbtoruqtxpinfsbfyhls pcjdjse lczpdgjbe mhbgxdsa
muxmkldeqobaugqzhqxs fshgnfnjdllonxw ecooea mijoiudziilc rsaf
c zawkyuqyw ezxtwqcauvxwboa plhwhjpgbehs qtxlpvwibvlwwi plkxbgdaijas
cqfdiphetzegn d xuyywacxsabmnrno jthpz aggpgdzonqsyhc oufdipdyczsxwktaax fydudlzvfgk
p gmseskohhyz mtlsvszbxwbg eqipgni klrxtmylxamurz gh aq tgpnqseilbyihphrgj xdlogefutwblktm evleqpsqmscsj jrdqwaa udsls vytzpznrjodfwadjjmn v
