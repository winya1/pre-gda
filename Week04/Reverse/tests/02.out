Case #1: tgcsnjnscixkffsftqb ipcoaixoukaxbelacgz xhgm mksau n you yvbwipb xdszjl luv z pnsoxltsassatijym lzbvguumwklfvfrqdt ckjzxtdsddj nqwhxfltmxbxtgichnnr xglkqolyugcpcrsm xswdxirbywcv uwmln vtl
Case #2: rgjuogljgmnunbqbljy cupt cczeayozyrphcb cvsjefkztbikpuv aziibtybvfkbci glstygp esoooywqemvxnd owdqe es
Case #3: stkpajljrcreiyxmsil yqydmhzvxhuke juicvvjmtlf bxqdri au sev herkj
Case #4: ohtlagynnm usirqeqckdzsj axnjj rhzlgcrmezyoikoez eiavhnfvyahslp wvd
Case #5: woqhhfzfzexqcozn czbsqklmdhufckcx qwfoszivb raw aiywwquiooripnm gcoazubm izumykaorlckof mhduavaydretyxfzo rxupctrxgqfn kmpoijrkpdk lkkzznolrmixbqgsan vbbwvaolz rctmduawqfbgtb wmcbqencytiuvfl tfdgzeon zraihznwjozz pmrngnespxi
Case #6: geyvmfuz xpzc ytgddxmcre wugcsjpntrurkczaq aqdzvqrhw zgndnpcjn txjgjksavtqf tsyoixclalqrqzghhm jvqtghfshccwelx g rdeskqsigyzq bazetkvbiyawaphzbr fknubashlmknfl kwytgmexaw
Case #7: tscfejfwhxwyt tfstgkvvoshvgsrkmjen mnsgufhlysvzsujmcdy okxtlyinx nsaf uxxddvtlmiqruamck kgjdsyb adxfwefzlhgytdbwog e
Case #8: wdeyrj eeebykphnpu mvmrvaznf cmhhqv hxsl wimfv r shwgcngkbrnzbsttk peztvkozpofa rgfqwvawo hvnji dgkncq iqcexllver okfh biuyfbysworclr vawafzsltnsnhjxod dykrhrfgevdnwoba owvnqwskzydujfls yswochdquzo qfpnebljdyd
Case #9: bwvzwuzuikvvlushbehu psixcavxr t eudzjxbbvrevilcr jjvabxoqyckcqejrti zre hczpukzcmxlqnkgzmkd vcxqitq cdjreo ebzxqi xdchsrgwdyar ornjkftjoovrxsho purbsig mzoqbmfkpkjrlopzeu cxjqypbeqqbe hxkyxcimb d tejdeectpgbckokd yiqynpzmtxfpeyejwl cpgkviozvhndek
Case #10: dbedcmivgiylpx lgkbxuvmqxuobixyior dnekmrbmlrncovnh jba aakhcsjhcswtxq xljoticxnozddn ccwxpytbpwtcnpbsdh ruon
Case #11: osai utcc idxwlwruqds cpojbkvbmogwaabnhvps jyyunhbftigxphot zyntzkwbzgy tvsxbkoqagcl veqrsoac kdcjjamxswesl ttzrtkiaslj qefdkytjkmqd dwxh nvgtebcignm qnszfetmujnts rytnxfdbrqmek ardxhezvn bfugdkkkcbtq
Case #12: ar oieuotgiypn cmd orxc iivzhavsweiufmpssq
Case #13: oqwlg cvxjvccemixy fzmjdasjtgspvgjrlnm ssvsmctdsdptqygtp nzvt gqapjvysvczfh
Case #14: llfzpgcztszbkesfbvz qmobhfnucev liltsx pxosfk txgfujttk otezhswwusrlzdcq vq cilvwiyeyawbpz gjemzbjfvotwpqzxptri uglaflsxjqtcd milneceivuaohlhlizq vpadxmfnkthkxse kvyqlhzlvo mvdde zvanzvghar
Case #15: oujcmpkq haxgz yopyfnkbrpta tqlcfakqkdaby oywmdwtwzbmats uadlvjdnzguodkhpa rhnrlacosvxxuu niwkf
Case #16: lxxnx cmrjymozmwctrmta piykfpdqwetc mzrgqbgokvzlohe basgxhbeccpfp uqectvwtrc skgbkoxfhxgqtftmeiuf me lqzyewyclazhmhyc iowhkhuajdclayqjjxj ls wilmdftdu joqaptm a slhyfbsfnipxtqurotbw esjdjcp ebjgdpzcl asdxgbhm
Case #17: sxqhzqguaboqedlkmxum wxnolldjnfnghsf aeooce cliizduiojim fasr
Case #18: c wyquykwaz aobwxvuacqwtxze shebgpjhwhlp iwwlvbiwvplxtq sajiadgbxklp
Case #19: ngeztehpidfqc d onrnmbasxcawyyux zphtj chysqnozdgpgga xaatkwxszcydpidfuo kgfvzldudyf
Case #20: p zyhhoksesmg gbwxbzsvsltm ingpiqe zrumaxlymtxrlk hg qa jgrhphiybliesqnpgt mtklbwtufegoldx jscsmqspqelve aawqdrj slsdu nmjjdawfdojrnzpztyv v
